vendor/github.com/jacobsa/gcloud/gcs/bucket.go

```diff
@@ -48,8 +48,8 @@ type Bucket interface {
 	// Official documentation:
 	//     https://cloud.google.com/storage/docs/json_api/v1/objects/get
 	NewReader(
-		ctx context.Context,
-		req *ReadObjectRequest) (io.ReadCloser, error)
+			ctx context.Context,
```

Comment:
Please remove unwanted changes from the cl

Reply:
this is the original code from master branch only

Reply:
if there is no change in the code, it shouldn't appear in the PR. There are some formatting changes in the cl. please fix.